service/src/test/java/dk/dbc/ess/service/EssServiceIT.java

```diff
@@ -465,6 +465,30 @@ public void baseNotValidTest() throws Exception {
         assertEquals(500, response.getStatus());
     }
 
+    @Test
+    public void maxPageSizeProperMetaRequest() throws Exception {
+        // Stubbing request to base
+        stubFor(get(urlEqualTo("/bibsys?query=horse&startRecord=1&maximumRecords=5"))
+                .willReturn(aResponse()
+                        .withStatus(200)
+                        .withHeader("Content-Type","text/xml")
+                        .withBodyFile("response_max_page_5.xml")));
+        stubFor(post(urlEqualTo("/"))
+                // Check root is format request with correct namespace
+                .withRequestBody(matchingXPath("//fr:formatRequest")
+                        .withXPathNamespace("fr","http://oss.dbc.dk/ns/openformat"))
+                .willReturn(aResponse()
+                        .withStatus(200)
+                        .withHeader("Content-Type","text/xml;charset=UTF-8")
+                        .withBodyFile("open_format_horse_response.xml")));
+        Response response = client.target(
+                String.format("http://localhost:%d/api/?base=bibsys&query=horse&format=netpunkt_standard&maximumRecords=100", dropWizzardRule.getLocalPort()))
```

Comment:
maximumRecords skal være rows.

Desuden kan du indføre en test specifikt for manglende rows parameter